mathesar/exceptions/error_codes.py

```diff
@@ -0,0 +1,13 @@
+from enum import Enum, unique
```

Comment:
I think this should all live within `mathessar/api`, since these are only API related exceptions.